README.md

```diff
@@ -90,42 +88,64 @@ The [**State-fold Server**](./offchain/state-server/README.md) is the component
 
-## Workflows
+## Framework features
 
-The Cartesi Node may act as a *User* (aka *Reader*) or *Validator* Node, hence performing different roles, which may be represented by different workflows.
+The Cartesi Node contains 3 main features, the combination of which defines the node capabilities
 
-### Reader Flow
+### Reader Feature
 
-In the Reader Flow, any *Input* read from the blockchain by the **State-fold Server** is received by the **Dispatcher** and relayed to the **Broker** through an input-specific stream.
+With the Reader Feature, any *Input* read from the blockchain by the **State-fold Server** is received by the **Dispatcher** and relayed to the **Broker** through an input-specific stream.
 The *Input* is eventually consumed by the **Advance Runner** and used to advance the state of the **Server Manager**, thus generating an *Advance* request to be processed by the underlying DApp.
 
 After finishing the processing, the DApp may generate a number of *Outputs*, which are eventually retrieved by the **Advance Runner** and fed back into the **Broker** through an output-specific stream.
 
 *Inputs* and *Outputs* are consumed from their respective streams by the **Indexer** and stored in a **PostgreSQL** database and may be queried by the DApp **Frontend** via a **GraphQL Server** instance.
 
-### Validator Flow
+### Claiming Feature
 
-The Validator Flow may only be performed by *Validator* nodes.
-It complements the Reader Flow by generating *Claims* at the end of *Epochs*, which are generated by the **Server Manager**, and sent to the **Broker** by the **Advance Runner** through a claims-specific stream to be eventually consumed by the **Dispatcher** for being submitted to the blockchain.
+The Claiming Feature allows a node to be run as a *Validator* node.
+It complements the Reader Feature by generating *Claims* at the end of *Epochs*, which are generated by the **Server Manager**, and sent to the **Broker** by the **Advance Runner** through a claims-specific stream to be eventually consumed by the **Authority Claimer** for being submitted to the blockchain.
 
-### Inspect Data Flow
+### Inspect Data Feature
 
 Every *Inspect* request sent by the **Frontend** via the Rollups HTTP API is captured by the **Inspect Server** and forwarded to the **Server Manager**, which queries the state of the underlying DApp via *Inspect* requests.
 
 Every request results in an *Inspect Response* that is returned to the **Inspect Server**, which sends it back to the **Frontend**.
 
-## Host mode
+## Execution modes
+
+The Cartesi Node may act as a *User* (aka *Reader*), as a *Validator* or run locally as a *Host*, hence performing different roles, which may be represented by different workflows.
+
+### Reader Mode
+
+The *Reader Mode* is the default running operation of the node. It doesn't submit claims to the blockchain, instead only reads and updates the state. It contains the Reader and Inspect Data features.
```

Comment:
> The *Reader Mode* is the default running operation of the node.

I'm afraid I have to disagree. We shouldn't provide a default mode because the user should be conscious of his actions.

Reply:
Fair enough, by default I meant that every node will run this logic, even a validator node that has the claiming as well. But I can see now I did not made myself clear.